js/contacts.js

```diff
@@ -43,19 +42,58 @@ document.addEventListener("DOMContentLoaded", function(event) {
                                 <div class="app-content-list-item-icon" style="">
                                 </div> 
                                 <div class="app-content-list-item-line-one" id="show_result" >
-                                    <p class="username-provider">There are no contacts!</p>
+                                    <p class="username-provider">There're no contacts!</p>
                                 </div>  
                             </div>`;                  
-                    var element = document.getElementById("show_result");
+                    var element = document.getElementById("test");
                     element.innerHTML = result;
                     $('#test_error').show();
 
                 }
             } 
         }
     }
+}).fail(function (response, code) {
+    console.log(response)
+    //alert('The token is invalid')
+});
+document.getElementById('elem').onclick = function () { 
```

Comment:
Why are you adding this here? What does it do?
I think this is code from accept.js that accidentally ended up in contacts.js
I removed it in https://github.com/pondersource/oc-sciencemesh/pull/35/files#diff-865e798ad64e5c4f552f82b73ce546ec3f2150da625ad60bb83db07c0e2d03ccL60-L68
But now you're adding it back, so that looks like you incorrectly resolved the merge conflict, probably?